glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 91a8d9a89a8e34c483b837ccd059cb40ebcea88394a9e6d9360425a749a0c7f2
```

Comment:
Yeah, its not intended, I was testing some other things and this got updated.  I will revert.